### PR TITLE
cephadm: bootstrap: nag about telemetry

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by liewegas** [comment]
retest this please

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -2024,7 +2024,8 @@ def command_bootstrap():
                     fsid,
                     args.output_config,
                     args.output_keyring))
-
+    logger.info('Please consider enabling telemetry to help improve Ceph:\n\n'
+                '\thttps://docs.ceph.com/docs/master/mgr/telemetry/\n')
```
Comment (sebastian-philipp) [inline src/cephadm/cephadm]: Instead of just pointing to the docs, we should also provide a **real** command to activate telemetry.

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Pull request merged**